### DISCUSSION TITLE
Linux Ubuntu client. Do we really need to use keyring?

### DISCUSSION THREAD

**Discussion by marek141017** [tutao/tutanota#6194] [General]
I have tried Linux appimage client and it is constantly trying to get access to my keyring.
Does it really necessary? 
Or can we use something like --password-store=basic to avoid it?

Thank you.

**Comment by ganthern** [comment]
That's not possible at the moment but the whole story around credential/key storage is going to change during the next releases. 
It's not impossible that there will be some way to pass through chromium's password-store cli argument.

We're using the keychain to store the secret used to access the data that's stored encrypted as part of your user data, namely saved credentials, the calendar alarms, offline cache and push identifier.

**Comment by marek141017** [comment]
Hello.
It is not possible even if one extracts files from appimage archive and execute binary?
It works for other extracted app images.
